ogre_mesh_viewer.py

```diff
@@ -223,6 +223,9 @@ def preRenderTargetUpdate(self, evt):
                     self.app._toggle_axes()
                 if ImGui.MenuItem("Show Bounding Box", "B", enode.getShowBoundingBox()):
                     self.app._toggle_bbox()
+                if ImGui.MenuItem("Polygon Mode", "M"):
```

Comment:
the UI aspect of this is not good. It is not obvious what mode you get into when you press the button.

I would rather make this a "Wireframe [ ]" toggle with checkmark as the others. Point rendering should not be that useful anyway..

Reply:
True, I tried using a submenu with the options but seemed to be too much trouble for the user.

This implementation is based on the legacy OGRE examples, but the suggestion is good